tools/accuracy_checker/accuracy_checker/dataset.py

```diff
@@ -208,6 +208,10 @@ def set_annotation_metadata(self, annotation, image, data_source):
         segmentation_mask_source = self.config.get('segmentation_masks_source')
         annotation.set_segmentation_mask_source(segmentation_mask_source)
         annotation.set_additional_data_source(self.config.get('additional_data_source'))
+        annotation.metadata.update({
```

Comment:
please use setter like above, there is annotation representation with additional logic - AnnotationContainer where process of updating meta different from regular

Reply:
and I believe you can just put full dataset meta, who knows maybe we need it in future